app/fields/file_field.rb

```diff
@@ -0,0 +1,7 @@
+require 'administrate/field/base'
+
+class FileField < Administrate::Field::Base
+  def file_url
```

Comment:
`file_url` seems like a confusing name here for what looks like a possibly-plural set of things that maybe aren't URLs (as suggested by the `files` in the next line, and the `each` in the `_show` partial below). What sort(s) of thing(s) is this function actually returning?

Reply:
Good catch. My initial plan was to return just, er, a `file_url` but the administrate thing didn't seem to work the way my brain was working so I just changed to what worked easily... and didn't change the name. I'll change it to `files` maybe as it returns all of the files associated with a single Thesis.